tests/client.py

```diff
@@ -455,6 +455,18 @@ def test_delete(self):
         self.solr.delete(q='*:*')
         self.assertEqual(len(self.solr.search('*:*')), 0)
 
+        # Try delete with a list of document ids
+        self.solr.add(self.docs)
+        self.solr.commit()
+        self.assertEqual(len(self.solr.search('*:*')), len(self.docs))
+        to_delete = [doc['id'] for doc in self.docs]
+        self.solr.delete(id=to_delete)
+        self.assertEqual(len(self.solr.search('*:*')), 0)
```

Comment:
I think this test would be safer if `to_delete` was all but one document and it checked to confirm that the single exception was not removed.